packages/create-contentful-app/template/src/components/Field.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Field from './Field';
+import { render } from '@testing-library/react';
+
+describe('Field component', () => {
+  it('Matches snapshot', () => {
+    const component = render(<Field />);
+
+    expect(component.container).toMatchSnapshot();
```

Comment:
Same here: I am not a big fan of snapshot testing. Maybe we should do a different assertion based on the component because I think it is very useful for the developer if we already have a solid test setup in place.

Reply:
Same as what @Jwhiles said above. The reason was to give a starting point for tests. We can change it to text assertion - as long as we retain some sort of starting point without being too opinionated.

Reply:
Changed it to text matching 🤷